Content/GUI/PassiveTree.cs

```diff
@@ -74,7 +82,12 @@ public override int InsertionIndex(List<GameInterfaceLayer> layers)
 
 	protected void DrawPanel()
```

Comment:
This and a few other methods are named a bit weirdly, we're not actually drawing anything and this is only called once when our UI is toggled on